app/models/statement.rb

```diff
@@ -3,7 +3,7 @@ class Statement < ApplicationRecord
   belongs_to :lead_provider
   has_many :statement_items
 
-  validates :output_fee, presence: true
+  validates :output_fee, inclusion: [true, false]
```

Comment:
should we add a test for this?

Reply:
Hi @cwrw Tests are in the other PR but I've added the same in this one as well.. tkss